webrender/res/cs_clip_rectangle.vs.glsl

```diff
@@ -18,8 +18,9 @@ struct ClipCorner {
     vec4 outer_inner_radius;
 };
 
-ClipCorner fetch_clip_corner(int index) {
-    vec4 data[2] = fetch_from_resource_cache_2(index);
+ClipCorner fetch_clip_corner(ivec2 address, int index) {
+    address += ivec2(2 + 2 * index, 0);
```

Comment:
... especially since we have the similar semantics here, where the offset is computed inside the `fetch*` function